config/packages/framework.yaml

```diff
@@ -0,0 +1,25 @@
+# see https://symfony.com/doc/current/reference/configuration/framework.html
+framework:
+    secret: '%env(APP_SECRET)%'
+    csrf_protection: true
+    http_method_override: true
+
+    # Enables session support. Note that the session will ONLY be started if you read or write from it.
+    # Remove or comment this section to explicitly disable session support.
+    session:
+        name:            packagist
+        cookie_lifetime: 3600
+        cookie_httponly: true
+        cookie_secure: auto
+        cookie_samesite: lax
+        handler_id: snc_redis.session.handler
+
+    #esi: true
+    fragments: true
+    php_errors:
+        log: true
+    form:            true
+    assets:
+        version: 'v=%env(default::ASSETS_VERSION)%'
+    templating:
```

Comment:
should be removed, as you don't depend on `symfony/templating`

Reply:
It seems to be needed for FOSUserBundle